### ISSUE TITLE
How to add using FetchContent or CPM?

### ISSUE THREAD

**Issue by adamski** [FigBug/Gin#48]
Is it possible to add a gin module via FetchContent or CPM?

I prefer this method over Git submodules.

**Comment by FigBug** [comment]
Maybe take a look at this project, they don't use submodules: https://github.com/stfufane/Midi-Transposer/blob/main/CMakeLists.txt

**Comment by stfufane** [comment]
Yeah it works like I did by using the `SOURCE_SUBDIR` argument to fetch the modules directory instead of the root folder, otherwise you get conflicts.
